src/_base/helm/app/templates/application/webapp/podmonitor.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.prometheus.podMonitoring -}}
+{{- if and (index .Values.docker.services "php-fpm" "enabled") .Values.prometheus.podMonitoring -}}
```

Comment:
Need to check for nginx or php-fpm due to lines 13-15

Reply:
nginx and php-fpm rather than or, as nginx just uses php-fpm for it's metrics. If a project just uses nginx then maybe can just overwrite the resource template